server/src/lib.rs

```diff
@@ -55,6 +56,7 @@ struct Claims {
 
 const EIP1559_ELASTICITY_MULTIPLIER: u64 = 6;
 const EIP1559_BASE_FEE_MAX_CHANGE_DENOMINATOR: U256 = U256::from_limbs([250, 0, 0, 0]);
+const TREASURY: AccountAddress = AccountAddress::ZERO; // todo what is the real address?
```

Comment:
I believe the treasury address is: `0x4200000000000000000000000000000000000011` based on https://docs.optimism.io/stack/transactions/fees#fee-collection-and-distribution

Reply:
+1 for making it configurable.